code/modules/antagonists/wizard/claw.dm

```diff
@@ -0,0 +1,67 @@
+/datum/antagonist/wizard/claw
```

Comment:
There's technically no reason for it to be a wizard subtype.

Reply:
wouldn't this ensure arbiter and claw work together? or does wiz subtype not do that

Reply:
Nope.